src/state/sandbox_exec.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh -ue
+
+POL='(version 1)(allow default)(deny network*)(deny file-write*)'
```

Comment:
is the default behaviour of the Linux one also to deny network connections?  That's a sensible default, but probably shouldn't diverge between operating systems

Reply:
yes, it's the same on Linux. There are now built-in mechanisms for package needing to download extra files, so better to enforce them.

Reply:
Thanks, I agree that these are excellent defaults.